util/package.go

```diff
@@ -159,7 +174,43 @@ func NewPackage(basePath string) (*Package, error) {
 		if p.PolicyTemplates[i].Multiple == nil {
 			p.PolicyTemplates[i].Multiple = &trueValue
 		}
+
+		// Collect basic information from policy templates and store into the /search endpoint
+		if p.PolicyTemplates != nil {
```

Comment:
I'm not sure about this condition if it's necessary. You're iterating over non-nil PolicyTemplates in for.

Reply:
removed, done!